### PR TITLE
ASSETS-17343 - Active user interface component lacks 3 to 1 contrast ratio.

### PR DESCRIPTION
**Pull request by TudorSMR**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by TudorSMR** [comment]
@paregupt @mijordan Can you please review this PR?

File: coral-component-checkbox/src/styles/index.styl [hunk 1]
```diff
@@ -27,3 +27,7 @@
 .coral--large ._coral-Checkbox-label {
   line-height: 1.6;
 }
+
+.coral--light ._coral-Checkbox-box:before {
+    border-color: #929292;
+}
```
Comment (majornista) [inline coral-component-checkbox/src/styles/index.styl:33]: These border colors should come from Spectrum-CSS. We should not be overriding color styles on an ad hoc, per-component basis.

**Review by majornista — CHANGES_REQUESTED** [review]

File: coral-component-checkbox/src/styles/index.styl (hunk 1 above)
Comment (majornista) [reply coral-component-checkbox/src/styles/index.styl:33]: Also, please verify that the border color in the version of Coral-Spectrum used in AEM is actually non-compliant. From what I can determine, the light theme checkbox border color is `rgb(142, 142, 142)` which has exactly 3:1 contrast ratio, against the standard coral-light background `rgb(245, 245, 245)`. Of course, this border color would become non-compliant when rendered against a slightly darker background, for example within a `Well` component.

The latest version of Spectrum-CSS has darker border colors for components: https://opensource.adobe.com/spectrum-css/checkbox.html, but I'm unsure of the roadmap for updating Coral-Spectrum to the latest version.

**Comment by paregupt** [comment]
I am not sure what this is about. You may want to remove me to protect this.

File: coral-component-checkbox/src/styles/index.styl (hunk 1 above)
Comment (TudorSMR) [reply coral-component-checkbox/src/styles/index.styl:33]: hi Michael. Here is a screenshot from the Jira issue, not sure if it's from 6.5 or 6.6, but I do remember checking with Color Contrast Analyzer and finding that it's not a 3:1 ratio.
image

About making changes to Spectrum-CSS, I had a conversation a while ago with Pareesh for a similar ticket where I was informed that it's an old version and we're not raising it at the moment, which is why I am going with this approach.

**Comment by TudorSMR** [comment]
@Pareesh adding the correct Pareesh :)

File: coral-component-checkbox/src/styles/index.styl (hunk 1 above)
Comment (majornista) [reply coral-component-checkbox/src/styles/index.styl:33]: The screenshot is showing CloudUI in 6.5. 6.6 uses Coral-Spectrum, which is using @adobe/spectrum-css@2.18.0, where the border color on checkboxes in the light theme is has exactly 3:1 contrast, which passes [WCAG SC 1.4.11 Non-text Contrast (AA)](https://www.w3.org/TR/WCAG22/#non-text-contrast).

Comment (TudorSMR) [reply coral-component-checkbox/src/styles/index.styl:33]: Hi @majornista  - you're right, I must have misused Color Contrast Analyzer when testing and have gotten invalid results. I just checked again and it looks like for 6.6 the contrast is ok, the issue is only in 6.5. Thanks for your help here, I will close this PR

**Pull request closed without merging**